appinventor/appengine/src/com/google/appinventor/client/TopPanel.java

```diff
@@ -267,7 +175,7 @@ public void showUserEmail(String email) {
    * Adds the MOTD box to the right panel. This should only be called once.
    */
   public void showMotd() {
```

Comment:
Do we know that this function is always called at most once? If not, there might be unexpected behavior adding the MOTD box multiple times.